wcag20/sources/techniques/css/C7.xml

```diff
@@ -30,8 +30,9 @@ a span {
 height: 1px;
 width: 1px;
 position: absolute;
+clip: rect(0 0 0 0);
+margin: -1px;
```

Comment:
This thread discusses it: https://github.com/h5bp/html5-boilerplate/issues/194  (in short, to avoid an errant single pixel of an item with a background color)